docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -73,22 +70,7 @@ function layout_builder_custom_form_alter(&$form, FormStateInterface $form_state
 
         if (isset($form[$lbs])) {
           $styles_exist = TRUE;
-
-          if ($form_id == 'layout_builder_update_block') {
-            // Set enabled flag based on whether a default value exists
-            // for this layout builder style.
-            $styles_enabled = !empty($form[$lbs]['#default_value']);
-          }
-
-          // Set a state for the layout builder style to toggle
-          // based on the value of the override value specified below.
-          $form[$lbs]['#states'] = [
-            'visible' => [
-              ':input[name="layout_builder_block_override"]' => [
-                'checked' => TRUE,
-              ],
-            ],
-          ];
+          $form[$lbs]['#attributes']['class'][] = 'layout-builder-style';
```

Comment:
@richardbporter Just curious why this was added. I don't see a corresponding style rule or `#states` that would seem to necessitate this.

Reply:
I'm now reading your comment above regarding this.

Reply:
Yea I thought we may want to add admin styles to make these look different. It could certainly be removed. Maybe a @GaryRidgway or @bspeare have ideas?